### PR TITLE
fix(table): Allow renaming column support.

### PR DESCRIPTION
**Pull request by jalcine**
Support for renaming columns was added in 3.25.0

Release Notes: https://sqlite.org/releaselog/3_25_0.html

Let me know if there's something off with it.

### CODE REVIEW

**Comment by ConnorRigby** [comment]
Ah i seem to have broken CI recently. I'm great with merging, i will just have to  Fix CI. Will be sure to merge tomorrow. :+1:

**Comment by jalcine** [comment]
Thanks! I was so excited to see this in the SQLite docs.

File: test/sqlite_ecto_test.exs [hunk 1]
```diff
@@ -1265,16 +1265,12 @@ defmodule Sqlite.Ecto2.Test do
 
   test "rename column" do
     rename = {:rename, table(:posts), :given_name, :first_name}
-    assert_raise ArgumentError, "RENAME COLUMN not supported by SQLite", fn ->
-      execute_ddl(rename)
-    end
+    assert execute_ddl(rename) == [~s|ALTER TABLE "posts" RENAME COLUMN "given_name" to "first_name"|]
```
Comment (ConnorRigby) [inline test/sqlite_ecto_test.exs]:
```suggestion
    assert execute_ddl(rename) == [~s|ALTER TABLE "posts" RENAME COLUMN "given_name" TO "first_name"|]
```

File: test/sqlite_ecto_test.exs [hunk 2]
```diff
@@ -1265,16 +1265,12 @@ defmodule Sqlite.Ecto2.Test do
...
   end
 
   test "rename column in prefixed table" do
     rename = {:rename, table(:posts, prefix: :foo), :given_name, :first_name}
-    assert_raise ArgumentError, "RENAME COLUMN not supported by SQLite", fn ->
-      execute_ddl(rename)
-    end
+    assert execute_ddl(rename) == [~s|ALTER TABLE "foo"."posts" RENAME COLUMN "given_name" to "first_name"|]
```
Comment (ConnorRigby) [inline test/sqlite_ecto_test.exs]:
```suggestion
    assert execute_ddl(rename) == [~s|ALTER TABLE "foo"."posts" RENAME COLUMN "given_name" TO "first_name"|]
```

**Comment by ConnorRigby** [comment]
@jalcine  Just ran tests manually, and those two up there fail. Looks like a typo. I suggested changes that should fix it.

**Comment by jalcine** [comment]
Cool. Looks like no failures but failed to upload coverage information.

**Comment by jalcine** [comment]
https://circleci.com/gh/elixir-sqlite/sqlite_ecto2/44 failed but it looks like it was only for coverage upload reasons.

**Comment by jalcine** [comment]
bumping @ConnorRigby

**Comment by ConnorRigby** [comment]
You are correct. I am not 100% sure what happened but i can get it manually merged in shortly.

**Pull request merged**